test/doe_prototype/test_small_office.rb

```diff
@@ -3,7 +3,7 @@
 
 class TestSmallOffice < CreateDOEPrototypeBuildingTest
   building_types = ['SmallOffice']
-  templates = ['DOE Ref Pre-1980','DOE Ref 1980-2004','90.1-2007','90.1-2013','90.1-2016','90.1-2019']
+  templates = ['DOE Ref Pre-1980','DOE Ref 1980-2004','90.1-2004','90.1-2007','90.1-2013','90.1-2016','90.1-2019']
```

Comment:
Let's maybe remove `90.1-2004` from the list so `templates` is consistent with the other building types.

Reply:
90.1-2004 is removed.